Tasks/ServiceFabricUpdateManifestsV2/Update-DockerImageSettings.psm1

```diff
@@ -61,7 +62,8 @@ function Update-DockerImageSettings
             }
             else
             {
-                $imageName = $imageDigestRepoName
+                $imageName = $imageDigestName
```

Comment:
Why do you need to change this line. In next line can you not just add to hash 
`$imageNameToDigestMapping[$imageDigestName] = $imageDigestValue`

Reply:
Done